src/kernel/emulation/linux/xattr/setattrlistat.c

```diff
@@ -0,0 +1,7 @@
+#include "setattrlistat.h"
+
+long sys_setattrlistat()
+{
+	return 0;
```

Comment:
Does this returning zero make it a silent error?

Maybe it should return an error code in case we encounter it again.

Reply:
If you return ```ENOTSUP```, ```ar``` will fail.

Reply:
OK, I suppose if something related to file attributes isn't working then someone will think to look there anyway